index.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const path = require('path');
+const fse = require('fs-extra');
+const findGitRoot = require('find-git-root');
+const contentRouter = require('./contentRouter');
+const contentTypesRouter = require('./contentTypesRouter');
+
+require('dotenv').config();
+
+async function server({ publicBaseUrl, dbDir }) {
+  const app = express();
+  const DEFAULT_PORT = 3020;
+  const port = parseInt(process.env.PORT, 10) || DEFAULT_PORT;
```

Comment:
process.env.PORT should be passed as a settings parameter or ...
maybe we should to think about a better interface to apply for flatlify server eventually.

A few options I could think of:
1. create a bin like `flatlify start -p 3000 -uploadDir ... -dbDir ...`
2. make it more of an express middleware 
3. having both above to provide more flexibility